namer/namer.cc

```diff
@@ -545,11 +605,14 @@ class SymbolDefiner {
                 return definedClasses[ref.idx()];
             case core::FoundDefinitionRef::Kind::Method:
                 ENFORCE(ref.idx() < definedMethods.size());
+                // Ensure that nobody is trying to reference alias methods.
+                ENFORCE(definedMethods[ref.idx()].exists());
```

Comment:
I didn't have a ruby snippet in mind when I wrote this.  I'm not exactly sure how it would come about.

But given that alias methods get nonexistent symbols in `definedMethods`, I was trying to make sure that we weren't referencing those alias methods for ownership.